conf/modules/align_MT.config

```diff
@@ -17,15 +17,20 @@
 
 process {
 
-    withName: '.*ALIGN_MT:BWAMEM2_MEM_MT' {
-        ext.when = { !(params.analysis_type == "wes") && params.aligner == "bwamem2" }
-        ext.args = { "-M -K 100000000 -R ${meta.read_group}" }
+    if (params.aligner.equals("bwamem2")) {
+        withName: '.*ALIGN_MT:BWAMEM2_MEM_MT' {
+            ext.args = { "-M -K 100000000 -R ${meta.read_group}" }
+            ext.when = { !(params.analysis_type.equals("wes")) }
```

Comment:
just because I am not so familiar with bwamem2, is there anything that prohibits us from using bwamem2 for wes for MT?